llvm/test/tools/llvm-cov/mcdc-macro.test

```diff
@@ -13,18 +13,18 @@
 // CHECK-NEXT:  |
 // CHECK-NEXT:  |  Number of Conditions: 5
 // CHECK-NEXT:  |     Condition C1 --> (9:7)
-// CHECK-NEXT:  |     Condition C2 --> (2:11)
-// CHECK-NEXT:  |     Condition C3 --> (3:11)
-// CHECK-NEXT:  |     Condition C4 --> (3:23)
-// CHECK-NEXT:  |     Condition C5 --> (9:22)
+// CHECK-NEXT:  |     Condition C2 --> (9:22)
+// CHECK-NEXT:  |     Condition C3 --> (2:11)
+// CHECK-NEXT:  |     Condition C4 --> (3:11)
+// CHECK-NEXT:  |     Condition C5 --> (3:23)
```

Comment:
`if (a && D && E || b)`

Why is the condition `b` above changed to be C2 rather than C5?  Intuitively it better matches the order of lines in the Branch Report coverage above as the 5th condition. I don't mind so much as long as it's consistent, but it seems less intuitive to me.

Reply:
> I could tweak the order to honor the location of `Expansion`. Your #78819 is better in this point. Is it better for me to rework?

That would be my preference. Honoring the location seems more intuitive to me when reading the output.